demos/random_cut_demo/src/main/scala/io/hydrosphere/rrcf/impl/RRCT.scala

```diff
@@ -207,24 +207,9 @@ class RRCT(initialData: Array[DataPoint], timeDecay: Int = 1000) {
     * @return
     */
   private[this] def locateDatapointInTree(dataPoint: DataPoint): Leaf = {

```

Comment:
Is this method necessary?
Since it's private, and used only after point was placed in reservoir, it's guaranteed that `get` returns `Some(leaf)`.
Moreover, you could remove possible exception.

Reply:
Method was usefull for debugging. Since all tests are passing now - it is obsolete